content/enterprise_influxdb/v1.10/introduction/installation/data_node_installation.md

```diff
@@ -127,7 +127,7 @@ For added security, follow these steps to verify the signature of your InfluxDB
 
 1. Download and import InfluxData's public key:
    ```
-   curl -s https://repos.influxdata.com/influxdb.key | gpg --import
+   curl -s https://repos.influxdata.com/influxdata-archive.key | gpg --import
```

Comment:
Here and elsewhere, for maximum compatibility, this should be `influxdata-archive_compat.key`, not `influxdata-archive.key`. See the FAQ of the blog post for details (the updates for the `-oss` filename change are being rolled out presently).

(Also, thank you for proactively adjusted to remove `-oss` from the filename).